### DISCUSSION TITLE
Meter Widget needle

### DISCUSSION THREAD

**Discussion by vavventuri** [israel-dryer/ttkbootstrap#608] [General]
Your meter is an excellent piece of code. Thank you VERY much. I do have a question. We engineers and scientists are familiar with rotating needle ( maybe red ) type gauges like a standard  volt meter,  pressure gauge or older car speedometer with a graduated scale. Your meter doesn't have a scale of units . Is there a way to implement a needle and scale with your widget ? Or alternately, suggestions how to modify your code to create such a widget. There are many of us looking for such an  meter. Other meter widget authors have incompatible THREADING problems with tkinter, which make using them on a tkinter window pure torture.You seem to have avoided such problems.

**Comment by israel-dryer** [comment]
@vavventuri, you can achieve this using the existing properties of the meter.  Here is a simple demo that shows how you might achieve an "needle" like effect:

<img width="213" height="135" alt="image" src="https://github.com/user-attachments/assets/c9c06c74-0d57-46e7-96f0-620e534a679b" />

  
```python
  ttk.Meter(
      master=frame,
      metersize=180,
      padding=5,
      amounttotal=280,
      meterthickness=50,
      arcrange=180,
      arcoffset=-180,
      amountused=75,
      textright='°',
      subtext='heat temperature',
      wedgesize=2,
      bootstyle='danger',
      interactive=True
  )
```
